hydra.c

```diff
@@ -138,13 +145,11 @@ hydro_copy(int place, TreeWalkQueryHydro * input, TreeWalk * tw)
     input->Density = SPHP(place).Density;
 #ifdef DENSITY_INDEPENDENT_SPH
     input->EgyRho = SPHP(place).EgyWtDensity;
-    input->EntVarPred = EntropyPred(place);
-    input->DhsmlDensityFactor = SPHP(place).DhsmlEgyDensityFactor;
-#else
-    input->DhsmlDensityFactor = SPHP(place).DhsmlDensityFactor;
+    input->EntVarPred = pow(SPHP(place).Entropy, 1/GAMMA);
 #endif
+    input->DhsmlEOMDensityFactor = SPHP(place).DhsmlEOMDensityFactor;
 
-    input->Pressure = PressurePred(place);
+    input->Pressure = Pressure(place);
```

Comment:
I feel we want to add a prefix to these function(Pressure / Entropy) -- at least add a verb,  GetPressure() to avoid potential conflicts with variable names.

Reply:
Ok.